dm/dumpling/dumpling.go

```diff
@@ -46,6 +48,7 @@ type Dumpling struct {
 
 	dumpConfig *export.Config
 	closed     atomic.Bool
+	realdumper *export.Dumper
```

Comment:
maybe we can call this `core` like lightning https://github.com/pingcap/ticdc/blob/3a2d2c5a5e9eec0eed946c5dfbdba3ce7210e526/dm/loader/lightning.go#L60